backend/app/controllers/spree/admin/payment_methods_controller.rb

```diff
@@ -39,7 +39,7 @@ def update
         end
 
         if @payment_method.update(attributes)
-          ensure_current_store
+          ensure_current_store(@payment_method)
```

Comment:
`(byebug) @payment_method.object_id
36240
(byebug) @object.object_id
36240`

I think we would be safe to use `@object` or `@payment_method` here, I went with `ensure_current_store(@payment_method)` as it fits the contents around it.